firebase-firestore/src/test/java/com/google/firebase/firestore/spec/SpecTestCase.java

```diff
@@ -1185,6 +1185,8 @@ private void runSteps(JSONArray steps, JSONObject config) throws Exception {
         acknowledgedDocs.clear();
         rejectedDocs.clear();
       }
+    } catch (Exception e) {
```

Comment:
Please revert.

Reply:
Fixed it. This helps identifying what went wrong with spec test.